ScanCommand/src/main/java/io/ballerina/scan/internal/ScanToolConstants.java

```diff
@@ -27,8 +27,11 @@ public class ScanToolConstants {
     static final String SCAN_COMMAND = "scan";
     public static final String RESULTS_JSON_FILE = "scan_results.json";
     static final String BALLERINA_RULE_PREFIX = "B";
-    public static final String PATH_SEPARATOR = "/";
-    public static final String BALLERINA_ORG = "ballerina";
+    static final String PATH_SEPARATOR = "/";
+    static final String BALLERINA_ORG = "ballerina";
```

Comment:
Is this a `ballerina` module?. isnt it `ballerinai` or `ballerinax`.

Reply:
bal tools should be of the orgs `ballerina`, `ballerinax`. So far we have, 
1. edi tool - `ballerina`
2. copybook - `ballerina`
3. health - `ballerinax`

We can also use one of those. Since this does not really an external package, `ballerina` is fine IMO.

looping @MaryamZi for more insight.

Reply:
This constant was meant to be used to label the fully qualified rule identifiers for non-core rules introduced via compiler plugins, specifically compiler plugins developed by  Ballerina platform developers, so that the resulting core rules and external rules are represented as:
1. Core rule: `B101`
2. ballerina/io compiler plugin rule: `ballerina/io:B101`

As `ballerinai` as well as `ballerinax` compiler plugins are also created by Ballerina platform developers, they will be added as constants as well.

Reply:
After discussions with @shafreenAnfar, since the scan tool is not tied with an external product, it was decided that it should go under `ballerina` org.